modules/checking-mco-status.adoc

```diff
@@ -52,13 +70,55 @@ $ oc describe mcp worker
   Updated Machine Count:      3
 Events:                       <none>
 ----
++
+[NOTE]
+====
+If a node is being cordoned, the node is not included in the `Ready Machine Count`. It is included in the `Unavailable Machine Count`:
+
+.Example output
+[source,terminal]
+----
+...
+  Degraded Machine Count:     0
+  Machine Count:              3
+  Observed Generation:        2
+  Ready Machine Count:        2
+  Unavailable Machine Count:  1
+  Updated Machine Count:      3
+----
+====
+
+. To see each existing `MachineConfig` object:
++
+[source,terminal]
+----
+$ oc get machineconfigs
+----
++
+[source,terminal]
```

Comment:
You could add an ".Example output" line before this one.

Reply:
Outdated